packages/dcos-integration-test/extra/test_rlimit.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import uuid
+
+
+def test_if_rlimits_can_be_used(cluster):
+    '''This test verifies that rlimits can be used.
```

Comment:
You could prefer double quotes `"` for this docstring style comments. I see that more common in our code base.